releasenotes/notes/56529.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+issue:
+  - 56529
+releaseNotes:
+- |
+  **Added** CRL support for plugged-in CA, enabling Istio to watch for ca-crl.pem files and 
```

Comment:
Was there a previous release note in the last PR? I don't remember, but if there was, you don't need this

Reply:
There wasn't. It did not ask me last time. So not sure how does prow is validating it. But this time it did so I added release note.